mc/gui/breathing_notification.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import time
+from PyQt5 import QtCore
+from PyQt5 import QtWidgets
+from PyQt5 import QtGui
+import mc.mc_global
+import mc.model
+
+
+class BreathingNotification(QtWidgets.QFrame):
+    # close_signal = QtCore.pyqtSignal(list, list)
+    breathe_signal = QtCore.pyqtSignal()
+    close_signal = QtCore.pyqtSignal()
+    # wait_signal = QtCore.pyqtSignal()
+
+    def __init__(self):
+        super().__init__()
+
+        self.setWindowFlags(
+            QtCore.Qt.Dialog
+            | QtCore.Qt.WindowStaysOnTopHint
+            | QtCore.Qt.FramelessWindowHint
+            | QtCore.Qt.WindowDoesNotAcceptFocus
+        )
+        # -To avoid the window getting focus we need to set both QtCore.Qt.Dialog
+        #  and QtCore.Qt.WindowDoesNotAcceptFocus (setting QtCore.Qt.Popup +
+        #  QtCore.Qt.WindowDoesNotAcceptFocus doesn't work)
+
+        # | QtCore.Qt.WindowStaysOnTopHint
```

Comment:
Was this meant to stay commented out, or has it been accidentally left in the code?

Reply:
I can't recall :) It would be useful to have the window staying on top of the others, i think there may have been some issue because of this, how different hints work might depend on what "base" we choose (dialog, window, popup, etc) when setting the Window params

Reply:
Actually when i test it now it will stay on top even without the flag, this is different from what it was like before when i had set the window type to be a "popup"